packages/api/cms-api/src/page-tree/dto/page-tree-node.input.ts

```diff
@@ -37,10 +37,9 @@ export abstract class PageTreeNodeBaseCreateInput {
     @ValidateNested()
     attachedDocument: AttachedDocumentInput;
 
-    @Field({ nullable: true })
+    @Field({ defaultValue: false })
```

Comment:
Do we need a default value here as we now have a default value in code?

Reply:
yes, because otherwise this would be a breaking change

Reply:
You could leave it optional in the GraphQL schema as there's now a default value in code. Then it wouldn't be a change to the public API at all